hugegraph-server/hugegraph-dist/src/assembly/travis/install-rocksdb.sh

```
@@ -0,0 +1,72 @@
+    exit 1
+fi
+if [ ! -d "$SERVER_LIB" ]; then
+    echo "Error: SERVER_LIB dir not found: $SERVER_LIB" >&2
+    exit 1
+fi
+if [ ! -f "$SERVER_BIN/common-topling.sh" ]; then
+    echo "Error: common-topling.sh not found under: $SERVER_BIN" >&2
+    exit 1
+fi
+
+source "$SERVER_BIN/common-topling.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./common-topling.sh was not specified as input (see shellcheck -x).

(SC1091)